config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
+  resources :purchases
+  resources :groups
+  devise_for :users
+
+  authenticated :user do
+    root to: 'groups#index', as: :authenticated_root
+  end
```

Comment:
**OPTIONAL**

- [ ] When you add this ' before_action:authenticate_user!` flag to all of your controllers you don't need this statement. The reason being every request will be checked if the user is logged in

Reply:
@peterrobert, I still left the routes as it is but added `skip_before_action :authenticate_user!, only: %i[index]` so that I can get back to my home page because the `before_action :authenticate_user!` prevents the home page from showing.
`